scandipwa/base-theme
# [SWPWA-434] OG tags - Social metadata to be provided where appropriate

Base on https://github.com/scandipwa/base-theme/pull/544

* Integrated OG metatags into SWPWA, based on recently implemented `meta component` and `meta store`.

To test:
1. install [this](https://chrome.google.com/webstore/detail/ayima-page-insights/nbjdmeadccgbgmjhbahkgadffhpejgbd) extension
2. open any page and check extension for meta tags
3. in case it does not work, press `F12 > Elements` and check header for tags.
